cmd/destroy/remote.go

```diff
@@ -48,50 +48,42 @@ COPY --from=certs /etc/ssl/certs /etc/ssl/certs
 COPY --from=installer /app/bin/* /okteto/bin/
 COPY --from=okteto-cli /usr/local/bin/* /okteto/bin/
 
-{{range $key, $val := .OktetoBuildEnvVars }}
```

Comment:
we are removing the envs then?

Reply:
Based on my experiments, the remote `okteto deploy` also computes these envvars, so it doesn't look like they bring any value here. Are you aware of any optimization that skips the computation of the envvars if they are already defined?